WBooks/Library/MainMenuController.swift

```diff
@@ -48,6 +53,14 @@ class MainMenuController: UIViewController {
         _view.tableView.register(cell: BookCell.self)
         _view.tableView.backgroundColor = UIColor.clear
     }
+    
+    private func configureNavBar() {
+        self.tabBarController?.setNavigationBarTitle("NAVIGATION_BAR_TITLE_LIBRARY".localized(), font: UIFont.systemFont(ofSize: 18, weight: UIFont.Weight.medium), color: UIColor.white)
```

Comment:
You are adding this for all View Controllers, you could add this function in an UIViewControllerExtension